docs/reference/hardware/automotive-camera-module-variants.md

```diff
@@ -40,6 +40,17 @@ This document contains reference information on various properties of PXIe-148X
 | PXIe-1486 Serializer   | 8 output     | FPD-Link III | DS90UB953  | N/A          | 2          | FlexRIO FPD-LINK™ III 953 Serializer   |
 | PXIe-1486 SerDes       | 4 in / 4 out | FPD-Link III | DS90UB953  | DS90UB954    | 2          | FlexRIO FPD-LINK™ III 953/954 SerDes   |
 
+#### Block Diagram
+![1486 block diagram](/docs/images/PXIe-1486-block-dia.png)
```

Comment:
3 things:
1. for the 954: instead of C0 and C1, I would use CLK0 and CLK1 for consistency
2. I'm pretty sure the 953 only has GPIO 0 to 3
3. I'd remove the voltage input from the PLL since it's extraneous information
4. Same goes for listing VDDPLL and VDDDRV